src/js/service/ImportService.js

```diff
@@ -130,4 +130,68 @@
       return pskl.utils.FrameUtils.createFromImage(resizedImage);
     });
   };
+
+  /**
+   * @param {!pskl.model.Piskel} originalPiskel
+   * @param {!pskl.model.Piskel} additionalPiskel
+   * @returns {pskl.model.Piskel}
+   */
+  // Adds frames from an additionalPiskel to an originalPiskel, adjusting the size of
+  // both piskels to the largest dimensions of either piskel, and centering the content.
+  // This will only work when there is only one layer.
+  // Multi layer animations are disabled from the UI in code-dot-org/piskel
+  ns.ImportService.prototype.mergePiskels = function(originalPiskel, additionalPiskel) {
```

Comment:
Not sure if the style is different in Piskel code base, but in most of our JS, function level comments take the form:
```js
/**
  * This is where I summarize things about the function
  * @param param1
  * @returns {Piskel}
 */